src/main.py

```diff
@@ -147,7 +148,17 @@ def get_dataset_by_id(
     request_id = request.headers.get("X-Request-ID", None)
     logger.info("Received request for dataset with ID", data={"dataset_id": dataset_id}, request_id=request_id)
 
-    if request.headers["Accept"] == JSONLD or BROWSABLE:
+    if request.headers["Accept"] == CSV:
```

Comment:
Can you add a comment above this to explain what it is. eg "If someone is asking for a csv at the dataset level we return the latest version of the latest edition.

If we come back and read this at a future date that'll be a really handy thing to know.